app/frontend/actions.js

```diff
@@ -385,29 +433,30 @@ module.exports = ({setState, getState}) => {
       sendResponse: '',
       sendAmount: Object.assign({}, state.sendAmount, {
         fieldValue: e.target.value,
+        coins: parseCoins(e.target.value),
       }),
     })
     validateSendFormAndCalculateFee()
   }
 
   const sendMaxFunds = async (state) => {
     setState({calculatingFee: true})
-
-    const maxAmount = await wallet.getMaxSendableAmount(state.sendAddress.fieldValue)
-
-    if (maxAmount > 0) {
+    let maxAmount
```

Comment:
maxAmount is not used beyond the scope of try{} so you can safely declare it as 'const' inside the block instead of 'let' outside

Reply:
resolved during merging